modules/example/src/smithy4s/example/FooService.scala

```diff
@@ -11,9 +11,16 @@ import smithy4s.kinds.PolyFunction5
 import smithy4s.kinds.toPolyFunction5.const5
 import smithy4s.schema.Schema.unit
 
+/** The most basics of services
+  * GetFoo is its only operation
+  */
 trait FooServiceGen[F[_, _, _, _, _]] {
   self =>
 
+  /** Returns a useful Foo
+    * No input necessary to find our Foo
+    * The path for this operation is "/foo"
+    */
```

Comment:
**question:** would this also include parameter docs? I don't think we have an example of that

Reply:
With parameter docs here you mean an `input` on an operation for example? I could chuck a doc comment on `PutObject`in `example.smithy`

Reply:
Let's defer this to a later PR, as there is a little bit of nuance to it with regards to packed inputs.